src/suresh/While.java

```diff
@@ -0,0 +1,75 @@
+package suresh;
+
+public class While {
+
+	void Even(int start, int end) {
+		System.out.println( "Even Numbers are:" );
+		int Index =start;
+		while (Index<=end) {
+			
+			if (Index % 2 == 0) {
+				
+				System.out.println(Index);
+				Index++;
+		}
+		}
+		
+	}
+	
+	void Divisible(int start, int end) {
+		System.out.println( "Numbers divisble by 5:" );
+		int Index =start;
+		while (Index<=end) {
+			
+			if (Index % 5 == 0) {
+				
+				System.out.println(Index);
+				Index++;
+		}
+		}
+		
+	} 
+	void Odd(int start, int end) {
+		System.out.println( "Numbers divisble by 3 & 5:" );
+		int Index =start;
+		while (Index<=end) {
+			
+			if (Index % 3 == 0 && Index % 5 == 0) {
+				
+				System.out.println(Index);
+				Index++;
+		}
+		}
+		
+	} 
+	void RangeDivisible(int start, int end) {
+		System.out.println( "Numbers divisble by 7 & 13:" );
+		int Index =start;
+		while (Index<=end) {
+			
+			if (Index % 7 ==0  ) {
+				
+				System.out.println(Index +" is Divisible by 7");
+				Index++;
+				
+		}}
+		int Index1 =start;
```

Comment:
why do we need different while loops

Please refer: 
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/blob/main/src/brijesh/LoopAssignment8.java